docs/references/subcommands/release-group.md

```diff
@@ -5,34 +5,43 @@ This `fossa release-group` subcommand allows users to interact with FOSSA releas
 It has the following subcommands:
 
 - [`fossa release-group add-projects`](./release-group/add-projects.md)
-- [`fossa release-group create`](./release-group//create.md)
+- [`fossa release-group create`](./release-group/create.md)
+- [`fossa release-group create-release`](./release-group/create-release.md)
 - [`fossa release-group delete`](./release-group/delete.md)
 - [`fossa release-group delete-release`](./release-group/delete-release.md)
 
 See the pages linked above for more details.
 
+>NOTE: For all subcommands that allow adding projects to releases, the combination of `--project-locator` , `--project-revision`, and `--project-branch` are required to identify the project that will be added to the release group. Multiple occurrences of these arguements are accepted and are grouped together based on their order. For example, `fossa release-group [create, create-release, add-projects] --title example-release-group --release example-release --project-locator custom+1/example --project-locator custom+1/example2 --project-revision 1234 --project-revision 5678 --project-branch main --project-branch main ` has project groupings of (project locator: custom+1/example, project revision: 1234, project branch: main) and (project locator: custom+1/example2, project revision: 5678, project branch: main).
```

Comment:
This might just be me, but this order of arguments feels confusing. I'd probably write each group of arguments together, like this:

```
fossa release-group [create, create-release, add-projects] --title example-release-group --release example-release --project-locator custom+1/example --project-revision 1234 --project-branch main --project-locator custom+1/example2 --project-revision 5678  --project-branch main 
```

This is optional, but I think it would be less confusing to write it like this.